src/docs/reference/core/Configuration/README.md

```diff
@@ -54,6 +59,24 @@ In addition you can specify a `Tenant` setting by using the Tenant Name, in this
 }
 ```
 
+You can also configure options from code in the web project's `Startup` class as explained in the [ASP.NET documentation](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options). A lot of Orchard features have options support while they don't offer default static configuration behavior so for those you need to write your own configuration code.
```

Comment:
```suggestion
You can also configure `IOptions` from code in the web project's `Startup` class as explained in the [ASP.NET documentation](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options). 

A lot of Orchard Core features are configured through the admin UI with site settings stored in the database. If you wish to override the site settings, you can do this with your own configuration code.
```

Reply:
Sorry for not being clear. 
I just meant Orchard is now Orchard Core, and when the docs get parsed by mkdocs, they read better with a few more line breaks. So here's a suggestion for breaking it up a bit

Reply:
Sure. Though since the whole documentation website is about Orchard Core I see no harm in shortening it to just "Orchard" when we're mentioning it the tenth time :).